README.md

```diff
@@ -2,7 +2,25 @@
 
 UQCS is dedicated to providing a community for everyone. Community participants violating the following rules may be sanctioned or expelled from the community at the discretion of the executive committee.
 
-We do not tolerate harassment of community participants in any form. Harassment includes offensive comments related to gender or anything to do with it (identity, history, expression, non-binary gender, etc.), sexual orientation, disability, physical appearance, body size, race, religion, programming experience or background, sexual images and languages in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of community activities, spaces, or other events, unwelcome physical contact, and unwelcome sexual attention. Participants asked to stop any harassing behavior are expected to comply immediately.
+We do not tolerate harassment of community participants in any form. Harassment includes, but is not limited to:
+* offensive comments related to:
+  * gender or anything to do with it (identity, history, expression, non-binary gender, etc.);
```

Comment:
reword this to:

> gender (including identity, history or expression)

Reply:
That should probably be done in another pull request, given that this pull request is about formatting.

Reply:
I agree we should move language changes to another PR. @nicklambourne are you happy to resolve your changes?